feed/tests/test_workflowlevel2view.py

```diff
@@ -16,38 +16,39 @@ def setUp(self):
         self.factory = APIRequestFactory()
         self.tola_user = factories.TolaUser()
 
-    def test_list_workflowLlevel2_superuser(self):
-        request = self.factory.get('/api/workflowLlevel2/')
+    def test_list_workflowLevel2_superuser(self):
+        request = self.factory.get('/api/workflowLevel2/')
```

Comment:
The original URL is with lowercase

Reply:
You're right. Even though the function doesn't use the URL.